packages/chat/src/components/AgentMessage/index.tsx

```diff
@@ -0,0 +1,68 @@
+import '../../styles.css';
```

Comment:
we should avoid importing this here, for any global imports we should place them in the package's entrypoint rather than scattering them across modules

this file should only be imported in `src/ui.ts`